app/javascript/articles/components/__tests__/SaveButton.test.jsx

```diff
@@ -0,0 +1,87 @@
+import { h } from 'preact';
+import { fireEvent, render } from '@testing-library/preact';
+import { axe } from 'jest-axe';
+import { SaveButton } from '../SaveButton';
+
+it('should have no a11y violations', async () => {
```

Comment:
This test is for accessibility violations. Using [jest-axe](https://github.com/nickcolley/jest-axe), it checks to see if the component violates any accessibility issues that jest-axe is aware of. If there are any, the test will fail.

You can read a bit more about it in this post, [Changelog: Frontend Edition](https://dev.to/devteam/changelog-frontend-edition-30l7), which I wrote a while back in regards to the Preact upgrade for Forem.